CHANGELOG.rst

```diff
@@ -23,6 +23,13 @@ Fixed
 Security
 ========
 
+[1.2.2] - 2019-04-26
```

Comment:
Please check https://github.com/kytos/of_core/blob/master/kytos.json#L6 before changing the version number. The right version here is 1.3.